plugins/GSdx/GSRendererDX.cpp

```diff
@@ -664,21 +678,7 @@ void GSRendererDX::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sourc
 
 			dev->SetupOM(om_dssel, om_bsel, afix);
 
-			dev->DrawIndexedPrimitive();
-
-			if (m_env.COLCLAMP.CLAMP == 0 && /* hack */ !tex && PRIM->PRIM != GS_POINTLIST)
-			{
-				GSDeviceDX::OMBlendSelector om_bselneg(om_bsel);
-				GSDeviceDX::PSSelector ps_selneg(m_ps_sel);
-
-				om_bselneg.negative = 1;
-				ps_selneg.colclip = 2;
-
-				dev->SetupOM(om_dssel, om_bselneg, afix);
-				dev->SetupPS(ps_selneg, &ps_cb, m_ps_ssel);
-
-				dev->DrawIndexedPrimitive();
-			}
```

Comment:
The two parts you replaced with SendDraw are not exactly the same BTW (this one doesn't have the dev->SetupOM line, which is a few lines above instead).

Reply:
Ah you're right I didn't notice that one, I guess I'll drop this change.